debian/changelog

```diff
@@ -1,3 +1,9 @@
+firefox (104.0ubuntu1) UNRELEASED; urgency=medium
```

Comment:
```suggestion
firefox (104.0.1) jammy; urgency=medium
```

Reply:
That's what I get for using `dch` instead of doing it manually.